wagtail_localize_rws_languagecloud/rws_client.py

```diff
@@ -93,10 +97,13 @@ def create_project(
         if not self.is_authenticated:
             raise NotAuthenticated()
 
+        # Ensure the name doesn't include special characters
+        cleaned_name = safe_characters.sub("", name)
```

Comment:
question: unlikely to happen in reality, but what if the project name is all special characters? worth being extra safe here?

Reply:
In that case I believe RWS will return a 400 response which will eventually crash the management command. I don't think that's _terrible_ given the edge case.

Do we want to set a default if name becomes blank? I think that might be too smart and not what I would expect as a user of this package.

Maybe we could just add some logging to help with debugging if ever users encounter this issue?

```python
if cleaned_name.strip().length == 0:
    self.logger.error("Project name is blank. Please specify a project name. Note that LanguageCloud does not allow special characters (aside from `_` and `-`) in the project name.")
```

Or maybe preemptively raise an exception?

```python
if cleaned_name.strip().length == 0:
    raise ValueError("Project name is blank. Please specify a project name. Note that special characters are stripped because LanguageCloud does not allow special characters (aside from `_` and `-`) in the project name.")
```

Reply:
I love the idea of the validation. 
Feels the message should be around the cleaned up project name. Perhaps "The cleaned up project name is blank. Note that special characters are stripped because LanguageCloud does not allow special characters (aside from `_` and `-`) in the project name."

Reply:
If you raise an exception in the sync job it basically amounts to the same thing as logging it and its quite hard for anyone to change the name by that stage.

Maybe if we want the user to intervene in that case, we could clean the name and throw that error in the `LanguageCloudProjectSettingsForm` so it gets thrown at a point where the user can still edit it?

Reply:
Yes, thank you Chris, that is exactly what I meant (sorry, heavy head 🤒, I really should not be lookoing at github)

Reply:
Looking into it more, I realized that because the prefix is required, there will always be a project name even if the title of the page is all special characters.

I think this answers the original question and I don't think we need to do anything more in my opinion. Is that OK?